vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1376,50 +1371,18 @@ public void failure(VimeoError error) {
      *
      * @param callback the callback to be invoked when the request finishes
      */
-    public void fetchCurrentUser(@NotNull ModelCallback<User> callback) {
+    public void fetchCurrentUser(@NotNull VimeoCallback<User> callback) {
         fetchCurrentUser(callback, null);
     }
 
     /**
      * Fetches the currently authenticated user from the API
      *
-     * @param callback    the callback to be invoked when the request finishes
-     * @param fieldFilter the field filter to apply to the request
-     */
-    public void fetchCurrentUser(@NotNull ModelCallback<User> callback, @Nullable String fieldFilter) {
-        fetchContent(Vimeo.ENDPOINT_ME, CacheControl.FORCE_NETWORK, callback, fieldFilter);
-    }
-
-
-    /**
-     * A generic synchronous GET call. Unless implementing your own threading,
-     * use {@link #fetchContent(String, CacheControl, ModelCallback, String, Map, String)}
-     *
-     * @param uri           URI of the resource to GET
-     * @param cacheControl  Cache control type (includes max age and other cache policy information)
-     * @param query         Query string for hitting the search endpoint
-     * @param refinementMap Used to refine lists (generally for search) with sorts and filters
-     * @param fieldFilter   The string of fields to include in the response (highly recommended!)
-     * @return the Retrofit {@link retrofit2.Response} object
+     * @param callback the callback to be invoked when the request finishes
+     * @param filter   the field filter to apply to the request
      */
-    @Nullable
-    public retrofit2.Response<Object> fetchContentSync(String uri, CacheControl cacheControl,
-                                                       @Nullable String query,
-                                                       @Nullable Map<String, String> refinementMap,
-                                                       @Nullable String fieldFilter) {
-        if (uri == null || uri.isEmpty()) {
-            throw new AssertionError("uri cannot be null or empty");
-        }
-        String cacheHeaderValue = createCacheControlString(cacheControl);
-
-        Map<String, String> queryMap = createQueryMap(query, refinementMap, fieldFilter);
-
-        Call<Object> call = mVimeoService.GET(getAuthHeader(), uri, queryMap, cacheHeaderValue);
-        try {
-            return call.execute();
-        } catch (IOException e) {
-            return null;
-        }
+    public void fetchCurrentUser(@NotNull VimeoCallback<User> callback, @Nullable String filter) {
```

Comment:
Consistency is good 👍 .   I'm with @kevinzetterstrom rename any `fetchXXXX` -> `getXXXX`.